cryptomator/src/main/java/ch/cyberduck/core/cryptomator/impl/CryptoFilenameV7Provider.java

```diff
@@ -23,7 +23,7 @@
 
 public class CryptoFilenameV7Provider implements CryptoFilename {
 
-    private static final int NAME_SHORTENING_THRESHOLD = 220; // https://github.com/cryptomator/cryptofs/issues/60#issuecomment-523238303
+    public static final int NAME_SHORTENING_THRESHOLD = 220; // https://github.com/cryptomator/cryptofs/issues/60#issuecomment-523238303
```

Comment:
Unrelated change?

Reply:
No, is config parameter in `vault.cryptomator`.